src/otlp.rs

```diff
@@ -0,0 +1,220 @@
+// Originally retired from davidB/tracing-opentelemetry-instrumentation-sdk
+// which is licensed under CC0 1.0 Universal
+// https://github.com/davidB/tracing-opentelemetry-instrumentation-sdk/blob/d3609ac2cc699d3a24fbf89754053cc8e938e3bf/LICENSE
+
+use std::{collections::HashMap, str::FromStr};
+
+use opentelemetry::trace::TraceError;
+use opentelemetry_otlp::SpanExporterBuilder;
+use opentelemetry_sdk::{
+    trace::{Sampler, Tracer},
+    Resource,
+};
+// #[cfg(feature = "tls")]
+// use tonic::transport::ClientTlsConfig;
```

Comment:
Do we want to support "tls"?

Reply:
If not, then maybe we should cleanup those commented blocks.

Reply:
Though, we could keep it like this for now and decide what to do with the TLS support later.